crates/runtime/src/datafusion.rs

```diff
@@ -124,8 +122,3 @@ impl Default for DataFusion {
         Self::new()
     }
 }
-
-#[must_use]
-pub fn get_internal_dataset_name(dataset: &str) -> String {
```

Comment:
We should probably have some verification code that the dataset names are valid so the user knows if they've included an invalid char in the name, especially now that the name can be whatever the user likes. In my experience the datafusion errors aren't very clear..